tests/begin_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2015 The appc Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"io/ioutil"
+	"path"
+	"testing"
+
+	"github.com/appc/acbuild/Godeps/_workspace/src/github.com/appc/spec/aci"
+	"github.com/appc/acbuild/Godeps/_workspace/src/github.com/appc/spec/schema"
+)
+
+func TestBegin(t *testing.T) {
+	tmpdir, err := setupTest()
+	if err != nil {
+		t.Fatalf("%v", err)
+	}
+
+	files, err := ioutil.ReadDir(path.Join(tmpdir, ".acbuild", "currentaci", aci.RootfsDir))
+	if err != nil {
+		t.Errorf("%v", err)
+	}
+	if len(files) != 0 {
+		t.Errorf("rootfs in aci contains files, should be empty")
+	}
+
+	manblob, err := ioutil.ReadFile(path.Join(tmpdir, ".acbuild", "currentaci", aci.ManifestFile))
+	if err != nil {
+		t.Errorf("%v", err)
+	}
+
+	man := &schema.ImageManifest{}
+
+	err = man.UnmarshalJSON(manblob)
+	if err != nil {
+		t.Errorf("invalid manifest schema: %v", err)
+	}
+
+	if len(man.Labels) != 0 {
```

Comment:
In this case I expect the slice of labels to be empty. The documentation for `reflect.DeepEqual` reads `An empty slice is not equal to a nil slice`. In your example, would my `w` be a nil slice or an empty slice?

All of the following comparisons result in the test failing:
```
if g := man.Labels; !reflect.DeepEqual(g, types.Labels{}) {
if g := man.Labels; !reflect.DeepEqual(g, nil) {
if g := man.Labels; !reflect.DeepEqual(g, make(types.Labels, 0)) {
```